docs.config.js

```diff
@@ -18,8 +18,6 @@ const webpack = config => {
 };
 
 module.exports = () => ({
-  docs: './packages/website/docs',
-  packages: ['./packages/*'],
```

Comment:
Did we remove these because these are the defaults? If so, can we remove the useManifests and webpack as well?